### ISSUE TITLE
i18n support

### ISSUE THREAD

**Issue by verlanmar** [satnaing/astro-paper#1]
Hi!

Would love to see the internationalization support, please.

Example implementation in Astro:

https://docs.astro.build

https://github.com/withastro/astro/tree/main/examples/docs

Thank you!

**Comment by satnaing** [comment]
Yep. It's a good feature and I'll try to add i18n support in the future releases. 
Thanks for your suggestion! ✌🏻

**Comment by satnaing** [comment]
Hello @verlanmar 

Sorry for my late response.
For internationalization support, I found that there are some good third-party integrations like [astro-i18next](https://astro-i18next.yassinedoghri.com/). Since users can integrate i18n feat easily with those tools, I decided not to include i18n support out of the box in [AstroPaper](https://astro-paper.pages.dev/). Hope you understand my decision. 

Thanks!
